### PR TITLE
Support paths as positional argument

### LINKED ISSUE #82: Run on a specific file only
We'd like to run `multilint` only on the changed files to test regression only (while we cleanup the rest of the code base).  Would it be possible to add positional arguments to the command that would accept file paths (and override `settings['paths']`)?

```shell
multilint path/file1.py path/file2.py
```

If you are ok with the idea, I'd be happy to submit a PR.

### PR DESCRIPTION
**Pull request by arthurio**
Fixes #82

### CODE REVIEW

File: multilint/__init__.py [hunk 1]
```diff
@@ -53,8 +60,13 @@ def main():
 
 def run(raw_args):
     settings = load_settings()
+
     args = parser.parse_args(raw_args)
     skip = args.skip or ()
+    paths = list(args.paths)
+
+    if paths:
+        settings['paths'] = paths
```
Comment (adamchainz) [inline multilint/__init__.py]: I'd prefer it if we went the other way and set `paths` from `settings['paths']` if `args.paths` is empty, then passed `paths` in below. This way we aren't mutating `settings`.

**Review by adamchainz — CHANGES_REQUESTED** [review]
Looking good, please also add a note in HISTORY.rst and a test

**Comment by adamchainz** [comment]
(Test edit `tox.ini` and add a line like `multilint setup.py multilint/`)

**Review by adamchainz — APPROVED** [review]

**Comment by adamchainz** [comment]
Released in version 2.4.0! https://pypi.org/project/multilint/
Thanks, enjoy.

**Comment by arthurio** [comment]
Thanks for the quick turnaround!

**Pull request merged**